### PR TITLE
creating Team section+styling it

### LINKED ISSUE #6: the-team-section
This team section includes:
-  "Meet the team" title
- short descriptive sentence.
- four cards for every team-members

### PR DESCRIPTION
**Pull request by Ibrahim-Jarada**
Relates #6

### CODE REVIEW

**Review by sara219 — CHANGES_REQUESTED** [review]
solve the conflict 
make sure that you have the latest ver of the main

**Review by ahmedhamouda2 — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -90,5 +92,28 @@ <h1>Projects</h1>
...
+          <figure>
+              <img class="team-images" src="./img/team/ahmed.jpg" alt="Ahmed Hamouda">
+              <figcaption>Ahmed Hamouda</figcaption>
+          </figure>
+          <figure>
+              <img class="team-images" src="./img/team/abdulrahman.jpg" alt="Abdelrahman Hassan">
+              <figcaption>Abdelrahman Hassan</figcaption>
+          </figure>
+          <figure>
+              <img class="team-images" src="./img/team/ibrahim.jpg" alt="Ibrahim Jarada">
```
Comment (sara219) [inline index.html:110]: the image didn't appear correctly 
you write `ibrahim`, and the image saved as `Ibrahim`

File: index.html [hunk 2]
```diff
@@ -90,5 +92,28 @@ <h1>Projects</h1>
...
+          <figcaption>Ibrahim Jarada</figcaption>
+            </figure>
+            <figure>
+                <img class="team-images" src="./img/team/ola.jpg" alt="Ola Ahel">
```
Comment (sara219) [inline index.html:114]: same goes here
`Ola` not `ola`

File: index.css [hunk 3]
```diff
@@ -327,3 +327,32 @@ hr {
...
+    background-size:cover;
+    background-repeat: no-repeat;
+    color: white;
+    text-align: center;
+
+}
+
+@media screen and (max-width : 768px){
+.team-images-container{
+    /* flex-direction: column; */
```
Comment (sara219) [inline index.css]: delete it if you don't want it

File: index.css [hunk 4]
```diff
@@ -327,3 +327,32 @@ hr {
 .clearix {
     clear: both;
 }
+ /* Ibrahim ============================================ */
```
Comment (sara219) [inline index.css]: add descriptive comment 

> our team section style

File: index.html [hunk 5]
```diff
@@ -90,5 +92,28 @@ <h1>Projects</h1>
         <img src="img/project/12.jpg" alt="project12">
     </section>
     <!-- End section projects -->
-    </body>
+     <section class="team-section">
```
Comment (sara219) [inline index.html:96]: add a comment, a descriptive one to tell us what's that section

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by sara219 — APPROVED** [review]

**Review by ahmedhamouda2 — APPROVED** [review]

**Review by Ola014 — APPROVED** [review]

**Pull request merged**